package.json

```diff
@@ -49,6 +49,7 @@
     "jscs-jsdoc": "2.0.0",
     "load-grunt-tasks": "3.5.0",
     "mkdirp": "0.5.1",
+    "mocha": "^3.2.0",
     "mocha-text-cov": "0.1.1",
     "nock": "8.0.0",
     "npmshrink": "1.0.1",
```

Comment:
i guess this needs to be removed? are we keeping the shrinkwrap file in the repo? if we are it might be useful to still copy that file into the docker image